modules/jooby-jetty/src/main/java/io/jooby/jetty/Jetty.java

```diff
@@ -90,24 +98,45 @@
       this.server = new Server(executor);
       server.setStopAtShutdown(false);
 
+      Http2Configurer<HttpConfiguration, List<ConnectionFactory>> http2;
+      if (options.isHttp2() == null || options.isHttp2() == Boolean.TRUE) {
```

Comment:
~~This means basically that if the extension present, then the flag does not need to be set explicitly but you have the ability to explicitely disable HTTP 2?~~

Reply:
Ok I found the answer in the docs.. :)